app/javascript/packages/normalize-yaml/visitors/collapse-sentence-spacing.js

```diff
@@ -0,0 +1,7 @@
+export default /** @type {import('yaml').visitor} */ ({
+  Scalar(_key, node) {
+    if (typeof node.value === 'string') {
+      node.value = node.value.replace(/(\w)\. {2,}/g, '$1. ');
```

Comment:
QQ how come we need the capture group? My expectation is that we could just grab the spaces and period?
```suggestion
      node.value = node.value.replace(/\. {2,}/g, '. ');
```

Reply:
> QQ how come we need the capture group? My expectation is that we could just grab the spaces and period?

In my mind I was thinking the capture group helped assure that it was actually a sentence of English text vs. some garbled symbols or some other non-"sentence" content like a numbered list, but (a) that's probably unlikely and (b) we may want the spacing collapse for those scenarios anyways?

Reply:
Yeah I agree we'd want to collapse anyways, and we could make the rule smarter if we run into issues in the future as needed?

Reply:
Maybe it's safe to even go so far as to collapse all multiple spaces everywhere, regardless of whether it occurs after a period? The resulting diff from existing issues seems reasonable.

```diff
diff --git a/app/javascript/packages/normalize-yaml/visitors/collapse-sentence-spacing.js b/app/javascript/packages/normalize-yaml/visitors/collapse-sentence-spacing.js
index 1ac577e9af..e4f5eaf8b1 100644
--- a/app/javascript/packages/normalize-yaml/visitors/collapse-sentence-spacing.js
+++ b/app/javascript/packages/normalize-yaml/visitors/collapse-sentence-spacing.js
@@ -3,3 +3,3 @@ export default /** @type {import('yaml').visitor} */ ({
     if (typeof node.value === 'string') {
-      node.value = node.value.replace(/(\w)\. {2,}/g, '$1. ');
+      node.value = node.value.replace(/ {2,}/g, ' ');
     }
diff --git a/config/locales/titles/fr.yml b/config/locales/titles/fr.yml
index ed8e21acab..c79bac9bf3 100644
--- a/config/locales/titles/fr.yml
+++ b/config/locales/titles/fr.yml
@@ -47,3 +47,3 @@ fr:
       suggest_second_mfa: Vous avez ajouté votre première méthode d’authentification !
-        Ajoutez-en une deuxième  en guise de sauvegarde.
+        Ajoutez-en une deuxième en guise de sauvegarde.
     no_auth_option: Aucun message de connexion trouvé
diff --git a/config/locales/user_mailer/en.yml b/config/locales/user_mailer/en.yml
index 0a68b4b203..aaa84453c1 100644
--- a/config/locales/user_mailer/en.yml
+++ b/config/locales/user_mailer/en.yml
@@ -285,3 +285,3 @@ en:
       gpo_letter_header: Your letter is on the way
-      header: To finish resetting your password, please click the link below  or copy
+      header: To finish resetting your password, please click the link below or copy
         and paste the entire link into your browser.
diff --git a/config/locales/zxcvbn/fr.yml b/config/locales/zxcvbn/fr.yml
index cea1f76730..01666fb39f 100644
--- a/config/locales/zxcvbn/fr.yml
+++ b/config/locales/zxcvbn/fr.yml
@@ -28,3 +28,3 @@ fr:
         Les répétitions comme « abcabcabc » sont à peine
-                 plus difficiles à deviner que « abc »
+         plus difficiles à deviner que « abc »
       reversed_words_arent_much_harder_to_guess: Les mots inversés ne sont pas très difficiles à deviner
```

Reply:
Gave that ⬆️ a try in rebased 0391204fae

Reply:
:shipit: